tests/test_sft_trainer.py

```diff
@@ -403,6 +446,46 @@ def test_run_causallm_lora_and_inference(request, target_modules, expected):
         assert "Simply put, the theory of relativity states that" in output_inference
 
 
+def test_successful_lora_target_modules_default_from_main():
+    """Check that if target_modules is not set, or set to None via JSON, the
+    default value by model type will be using in LoRA tuning.
+    We use MODEL_ARGS to run tuning, which has no target_modules set. During HF
+    training process, the correct default target modules will be used for model
+    type llama and will then exist in the resulting in adapter_config.json.
```

Comment:
nit: slight rewording for clarity and making a little more succinct
```suggestion
    """Check if target_modules is not set or set to None, the default
    target_modules by model type will be using in LoRA tuning.
    The correct default target modules will be used for model type
    LlamaForCausalLM and will exist in the resulting adapter_config.json.
```